.github/workflows/terraform-compatibility-matrix.yml

```diff
@@ -0,0 +1,61 @@
+name: "HashiCorp Terraform Compatibility Matrix"
+run-name: 'HashiCorp Terraform Compatibility Matrix ${{ inputs.atlas_cloud_env }}'
+
+on:
+  schedule:
+    - cron: "0 0 * * 6" # runs every Saturady at midnight UTC
+  workflow_dispatch:
+    inputs:
+      atlas_cloud_env:
+        description: 'Atlas cloud environment used, can be either `dev` or `qa`, empty for `dev`'     
+        type: string
+        required: false
+      test_group:
+        description: 'Test group to run, e.g. advanced_cluster, empty for all'
+        type: string
+        required: false
+      ref:
+        description: 'The branch, tag or SHA where tests will run, e.g. v1.14.0, empty for default branch'
+        type: string
+        required: false
+
+jobs:
+  get-supported-versions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+        with:
+          ref: ${{ inputs.ref || github.ref }}
+      - name: Get HashiCorp Terraform supported versions
+        shell: bash
+        id: get-terraform-supported-versions
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          output=$(./scripts/get-terraform-supported-versions.sh)
+          quoted_output=$(echo "$output" | jq -c .)
+          echo "supported_versions=$quoted_output" >> "$GITHUB_OUTPUT"
+      - name: Print HashiCorp Terraform supported versions
+        shell: bash
+        run: |
+          echo "${{ steps.get-terraform-supported-versions.outputs.supported_versions }}" 
```

Comment:
what is the purpose of this? why do we want to print it?

Reply:
Was for debugging purposes when developing, will remove it now